app/app.go

```diff
@@ -659,13 +679,17 @@ func (app *app) checkForUpdates(passive bool) {
 			log.Println("update: you have the latest version")
 		}
 		if !passive {
-			app.runtime.Dialog.Message(&dialog.MessageDialog{
+			_, err := app.runtime.Dialog.Message(&dialog.MessageDialog{
 				Type:         dialog.InfoDialog,
 				Title:        "You're up to date",
 				Message:      fmt.Sprintf("%s is the latest version.", latest.TagName),
 				Buttons:      []string{"OK"},
 				CancelButton: "OK",
 			})
+			if err != nil {
+				// @matryer - how do you want to handle these errors?
```

Comment:
not much we can do if these error is there?

Reply:
100% safe to ignore in this context. The error is mainly for text that cannot be converted to UTF-16.